### DISCUSSION TITLE
Are lists sortable?

### DISCUSSION THREAD

**Discussion by wizonesolutions** [vazco/uniforms#1070] [Q&A] [answered]
I'm looking to move to this package from autoform in an older Meteor project. The issue I am encountering there is that there's no easy way to have a sorted list field. I have a list of tasks as part of a parent object, and there is an `order` field on the tasks. When I click on up/down arrows, I want to move them up and down in real time based on the value of the field.

Is this achievable in uniforms?

**Comment by radekmie** [comment, marked as answer]
Hello @wizonesolutions! This one was already answered, e.g., in https://github.com/vazco/uniforms/issues/639#issuecomment-553446148. In general, it is, of course, possible. However, none of the provided themes have this option. You can create your custom `ListField` field and there implement sorting.

The point of not having it is that it's unclear _how_ it should look and feel. Depending on the list, it's better to have the rearrangement buttons on the side/on top/via drag and drop.
